src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
Hmm, I went down the rabbithole and finally arrived at `RequestParser::parseContent()` which does
```c++
m_request.posts[pair.first.toLower()] = pair.second;
```
And it doesn't convert to lowercase for the value. Maybe there are valid reasons not to do so...
I am waiting for someone else to comment.